multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/compose/elements/MarkdownText.kt

```diff
@@ -143,7 +145,7 @@ fun MarkdownText(
                 if (placeholderState.animate) animations.animateTextSize(it) else it
             },
         style = style,
-        inlineContent = mapOf(
+        inlineContent = inlineContent.inlineContent + mapOf(
```

Comment:
Since inlineContent.inlineContent + mapOf(...) creates a new map instance every time, it should be wrapped in remember to avoid unnecessary recompositions.

Reply:
Good point